packages/tools/replay-tool/src/replayMessages.ts

```diff
@@ -349,14 +349,17 @@ export class ReplayTool {
 
         // GC will consider unreferenced node in old documents to be inactive. When such nodes receive ops or are
         // loaded, GC will log an error. Ignore those errors since we don't care about them when replaying old docs.
-        if (event.eventName.includes("GarbageCollector:inactiveObject_")) {
+        if (event.eventName.includes("GarbageCollector:inactiveObject_")
+            // there is something wrong with summary ops, but we don't run the summarizer, so it shouldn't matter
+            || event.eventName.includes("SummaryAckWithoutOp")
```

Comment:
SummaryAckWithoutOp being logged when running SharedMatrixDeletes snapshot tests
we should figure out why this is being logged, fix it, and remove this supression

Reply:
I repro'd this locally and I think I know what is happening. The snapshot frequency for the stress test is 50 which means that after every 50 ops, it take a snapshot of the container, loads a new container from this snapshot and replays rest of the ops (among other things).
In the `messages.json` file that you added, op# 100 is `summarize` and op# 101 is `summaryAck`. So, at some point, a container loads from snapshot at op# 100 and sees a `summaryAck`. It thinks it got a summary ack without an op. If the snap point was an op above or below, you would not see this errror.